app/src/main/java/org/mozilla/focus/observer/AverageLoadTimeObserver.java

```diff
@@ -0,0 +1,38 @@
+package org.mozilla.focus.observer;
+
+import android.os.SystemClock;
+import android.util.Log;
+
+import org.mozilla.focus.architecture.NonNullObserver;
+import org.mozilla.focus.session.SessionManager;
+import org.mozilla.focus.telemetry.TelemetryWrapper;
+
+public class AverageLoadTimeObserver extends NonNullObserver<Boolean> {
```

Comment:
\[PMD\] INFO: The class 'AverageLoadTimeObserver' has a Cyclomatic Complexity of 5 \(Highest = 4\)\.